test/link-style-consistent.md

```diff
@@ -0,0 +1,21 @@
+# Consistent Link Style
+
+Text [url](https://example.com) text
+Text ![url](https://example.com) text
+Text [url] {MD054} text
+Text ![url] {MD054} text
+Text [text][url] {MD054} text
+Text ![text][url] {MD054} text
+Text <https://example.com> text
+Text [url][] text {MD054}
+Text [link text][url] text {MD054}
```

Comment:
This one seems like the same as like 7? And maybe still missing an example for `![url][]`?